RELEASE.md

```diff
@@ -0,0 +1,72 @@
+# Releases
+
+This page describes the release process and the currently planned schedule for upcoming releases as well as the respective release shepherd.
+
+## Release schedule
+
+| release series | date  (year-month-day) | release shepherd                            |
+|----------------|--------------------------------------------|---------------------------------------------|
+| v1.0.0           | 2021-04-22                                 | leiwanjun (GitHub: @leiwanjun) |
+| v1.2.0           | 2021-07-15                                | leiwanjun (GitHub: @leiwanjun) |
+| v1.3.0           | 2021-08-10                                 | zhu733756 (GitHub: @zhu733756) |
+
+# How to cut a new release
+
+> This guide is strongly based on the [Prometheus release instructions](https://github.com/prometheus/prometheus/blob/master/RELEASE.md).
+
+## Branch management and versioning strategy
+
+We use [Semantic Versioning](http://semver.org/).
+
+We maintain a separate branch for each minor release, named `release-<major>.<minor>`, e.g. `release-1.1`, `release-2.0`.
+
+The usual flow is to merge new features and changes into the main branch and to merge bug fixes into the latest release branch. Bug fixes are then merged into main from the latest release branch. The main branch should always contain all commits from the latest release branch.
+
+If a bug fix got accidentally merged into main, cherry-pick commits have to be created in the latest release branch, which then have to be merged back into main. Try to avoid that situation.
+
+Maintaining the release branches for older minor releases happens on a best effort basis.
+
+## Prepare your release
+
+For a new major or minor release, work from the `main` branch. For a patch release, work in the branch of the minor release you want to patch (e.g. `release-0.1` if you're releasing `v0.1.1`).
+
+<!-- Change the `Install the latest stable version` section in README.md to the new stable version:
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubesphere/notification-manager/master/config/bundle.yaml
+``` -->
+
+Add an entry for the new version to the `CHANGELOG.md` file. Entries in the `CHANGELOG.md` are meant to be in this order:
+
+* `[CHANGE]`
+* `[FEATURE]`
+* `[ENHANCEMENT]`
+* `[BUGFIX]`
+
+Create a PR for the changes to be reviewed.
+
+## Publish the new release
+
+For new minor and major releases, create the `release-<major>.<minor>` branch starting at the PR merge commit.
+From now on, all work happens on the `release-<major>.<minor>` branch.
+
+Bump the version in the `VERSION` file in the root of the repository.
+Build and push the container image:
+
+```bash
+make build
```

Comment:
If you take a look at the notification manager's makefile, the build and push of nm are a bit different, you can adjust this to :
```
Make sure you've set up docker buildx environment, then run the following command to build and push amd64 and arm64 images at the same time:
make build
```

Reply:
fixed